docs/streams/developer-guide/config-streams.html

```diff
@@ -1140,7 +1125,7 @@ <h4><a class="toc-backref" href="#id18">Default Values</a><a class="headerlink"
           <td>Consumer</td>
           <td><code class="docutils literal"><span class="pre">1000</span></code></td>
         </tr>
-        <tr class="row-odd">
+        <tr class="row-even">
```

Comment:
again, I think these row changes are from trunk

Reply:
Well,l the row above this one is "even" so this one should be "odd" -- no need to set it to "even"